Symfony/CS/Fixer/Symfony/NoEmptyCommentFixer.php

```diff
@@ -125,4 +133,41 @@ private function clearCommentToken(Tokens $tokens, $index)
         // override with whitespace token linebreak
         $tokens->overrideAt($index, array(T_WHITESPACE, "\n", $tokens[$index]->getLine()));
     }
+
+    private function isSurroundedBySingleLineComments(Tokens $tokens, $index)
+    {
+        $line = $tokens[$index]->getLine();
+        $prev = $tokens->getPrevNonWhitespace($index);
+        $next = $tokens->getNextNonWhitespace($index);
+
+        return $this->isSingleLineCommentOnLine($tokens, $prev, $line - 1) &&
+               $this->isSingleLineCommentOnLine($tokens, $next, $line + 1);
+    }
+
+    private function isSingleLineCommentOnLine(Tokens $tokens, $index, $line)
+    {
+        if ($index === null) {
+            return false;
+        }
+
+        $token = $tokens[$index];
+
+        if (false === $token->isComment()) {
+            return false;
+        }
+
+        if ($line !== $token->getLine()) {
```

Comment:
Turns out that the line number checks are not necessary at all. I've added tests to prove that in 1291ec9.